README.md

```diff
@@ -4,259 +4,240 @@
-- Read and Write support for object storage (AWS S3, Azure, Cloudflare R2, or Google GCS):
-	- Read parquet, CSV and JSON files:
-		- `SELECT * FROM read_parquet('s3://bucket/file.parquet')`
-		- `SELECT r['id'], r['name'] FROM read_csv('s3://bucket/file.csv') r`
-		- `SELECT count(*) FROM read_json('s3://bucket/file.json')`
-		- You can pass globs and arrays to these functions, just like in DuckDB
-	- Enable the DuckDB Iceberg extension using `SELECT duckdb.install_extension('iceberg')` and read Iceberg files with `iceberg_scan`.
-	- Enable the DuckDB Delta extension using `SELECT duckdb.install_extension('delta')` and read Delta files with `delta_scan`.
-	- Write a query — or an entire table — to parquet in object storage.
-		- `COPY (SELECT foo, bar FROM baz) TO 's3://...'`
-		- `COPY table TO 's3://...'`
-		- Read and write to Parquet format in a single query
-
-			```sql
-			COPY (
-				SELECT count(*), r['name']
-				FROM read_parquet('s3://bucket/file.parquet') r
-				GROUP BY name
-				ORDER BY count DESC
-			) TO 's3://bucket/results.parquet';
-			```
-- Read and Write support for data stored in MotherDuck
-- Query and `JOIN` data in object storage/MotherDuck with Postgres tables, views, and materialized views.
-- Create temporary tables in DuckDB its columnar storage format using `CREATE TEMP TABLE ... USING duckdb`.
-- Install DuckDB extensions using `SELECT duckdb.install_extension('extension_name');`
-- Toggle DuckDB execution on/off with a setting:
-	- `SET duckdb.force_execution = true|false`
-- Cache remote object locally for faster execution using `SELECT duckdb.cache('path', 'type');` where
-	- 'path' is HTTPFS/S3/GCS/R2 remote object
-	- 'type' specify remote object type: 'parquet' or 'csv'
-
-## Installation
+## See It In Action
 
-### Docker
-
-Docker images are [available on Dockerhub](https://hub.docker.com/r/pgduckdb/pgduckdb) and are based on the official Postgres image. Use of this image is [the same as the Postgres image](https://hub.docker.com/_/postgres/). For example, you can run the image directly:
+### Instant Analytics
 
-```shell
-docker run -d -e POSTGRES_PASSWORD=duckdb pgduckdb/pgduckdb:16-main
-```
+Transform your PostgreSQL into a data lake powerhouse:
 
-And with MotherDuck, you only need a [a MotherDuck access token][md-access-token] and then it is as simple as:
-```shell
-$ export MOTHERDUCK_TOKEN=<your personal MD token>
-$ docker run -d -e POSTGRES_PASSWORD=duckdb -e MOTHERDUCK_TOKEN pgduckdb/pgduckdb:16-main
+```sql
+-- Setup S3 access in seconds
+SELECT duckdb.create_simple_secret(
+    type := 'S3', key_id := 'your_key', secret := 'your_secret', region := 'us-east-1'
+);
+
+-- Query terabytes of Parquet data like local tables
+SELECT 
+    region, 
+    date_trunc('month', order_date) as month,
+    SUM(revenue) as monthly_revenue,
+    approx_count_distinct(customer_id) as unique_customers
+FROM read_parquet('s3://datalake/orders/year=2024/**/*.parquet')
+GROUP BY ALL;
```

Comment:
This syntax is invalid afaict. Same for the queries below.
```suggestion
SELECT 
    r['region'], 
    date_trunc('month', r['order_date']) as month,
    SUM(r['revenue']) as monthly_revenue,
    approx_count_distinct(r['customer_id']) as unique_customers
FROM read_parquet('s3://datalake/orders/year=2024/**/*.parquet') r
GROUP BY ALL;
```

Reply:
Or do read_parquet and read_csv always return a `duckdb.row` and so it's necessary to always use "r"?